source/crud/tests/write/updateOne-arrayFilters.yml

```diff
@@ -72,18 +72,18 @@ tests:
             arguments:
                 filter: {_id: 3}
                 update:
-                    $set: {y.$[i].c.$[j].d: 0}
+                    $set: {"y.$[i].c.$[j].d": 0}
                 arrayFilters:
                   - {i.b: 5}
                   - {j.d: 3}
 
         outcome:
             result:
                 matchedCount: 1
-                modifiedCount: 1
+                modifiedCount: 0 
```

Comment:
Copy-paste error; as the test description suggests, no document matches the array filters, so nothing is modified.